source/speech/speech.py

```diff
@@ -2072,7 +2072,14 @@ def getControlFieldSpeech(  # noqa: C901
 		# #10095, #3321, #709: Report the name and description of groupings (such as fieldsets) and tab pages
 		# #13307: report the label for landmarks and regions
 		nameAndRole = nameSequence[:]
-		nameAndRole.extend(roleTextSequence)
+		if (
+			role in (

```

Comment:
Sorry, I may have miscommunicated. The logic you have here is to speak the role if the role is landmark or region and reportLandmarks is true.
However, it should still continue to speak the role for grouping and property page as well.
So... 
```suggestion
			role not in (
```

Reply:
thanks, yes I was a bit confused by my interpretation of the suggested logic